lib/transcoder/video.ex

```diff
@@ -112,15 +115,16 @@ defmodule Membrane.Transcoder.Video do
     })
   end
 
-  defp maybe_plug_encoder_and_parser(builder, %VP8{}) do
+  defp maybe_plug_encoder_and_parser(builder, %vpx{}) when vpx in [VP8, VP9] do
     cpu_quota = :erlang.system_info(:cpu_quota)
 
     number_of_threads =
       if cpu_quota != :unknown,
         do: cpu_quota,
         else: :erlang.system_info(:logical_processors_available)
 
-    builder |> child(:vp8_encoder, %VP8.Encoder{g_threads: number_of_threads, cpu_used: 15})
+    encoder = Module.concat(vpx, Encoder) |> struct!(g_threads: number_of_threads, cpu_used: 15)
```

Comment:
I don't like calling `struct!/2` as it allows to create a struct that misses some required fields and I am not sure if it supports non-nil default values

Reply:
I removed `struct!()` usage for both H26x and VPx